CHANGELOG.md

```diff
@@ -74,6 +76,26 @@ librdkafka.
 
 ### Producer fixes
 
+ * Transactional producer could crash on request timeout due to dereferencing
```

Comment:
should these be in the v1.5.0 section given they actually arrived in v1.4.4 according to the section below?

Reply:
Good point!